modules/imaging_browser/js/imagingBrowserIndex.js

```diff
@@ -0,0 +1,2 @@
.instancePool.length<Klass.poolSize){Klass.instancePool.push(instance)}};var fourArgumentPooler=function fourArgumentPooler(a1,a2,a3,a4){var Klass=this;if(Klass.instancePool.length){var instance=Klass.instancePool.pop();Klass.call(instance,a1,a2,a3,a4);return instance}else{return new Klass(a1,a2,a3,a4)}};function MapBookKeeping(mapResult,keyPrefix,mapFunction,mapContext){this.result=mapResult;this.keyPrefix=keyPrefix;this.func=mapFunction;this.context=mapContext;this.count=0}MapBookKeeping.prototype.destructor=function(){this.result=null;this.keyPrefix=null;this.func=null;this.context=null;this.count=0};addPoolingTo(MapBookKeeping,fourArgumentPooler);function mapSingleChildIntoContext(bookKeeping,child,childKey){var result=bookKeeping.result;var keyPrefix=bookKeeping.keyPrefix;var func=bookKeeping.func;var context=bookKeeping.context;var mappedChild=func.call(context,child,bookKeeping.count++);if(Array.isArray(mappedChild)){mapIntoWithKeyPrefixInternal(mappedChild,result,childKey,emptyFunction.thatReturnsArgument)}else if(mappedChild!=null){if(React.isValidElement(mappedChild)){mappedChild=cloneAndReplaceKey(mappedChild,keyPrefix+(mappedChild.key&&(!child||child.key!==mappedChild.key)?escapeUserProvidedKey(mappedChild.key)+"/":"")+childKey)}result.push(mappedChild)}}function mapIntoWithKeyPrefixInternal(children,array,prefix,func,context){var escapedPrefix="";if(prefix!=null){escapedPrefix=escapeUserProvidedKey(prefix)+"/"}var traverseContext=MapBookKeeping.getPooled(array,escapedPrefix,func,context);traverseAllChildren(children,mapSingleChildIntoContext,traverseContext);MapBookKeeping.release(traverseContext)}var numericPropertyRegex=/^\d+$/;var warnedAboutNumeric=false;function createReactFragment(object){if(typeof object!=="object"||!object||Array.isArray(object)){warning(false,"React.addons.createFragment only accepts a single object. Got: %s",object);return object}if(React.isValidElement(object)){warning(false,"React.addons.createFragment does not accept a ReactElement "+"without a wrapper object.");return object}invariant(object.nodeType!==1,"React.addons.createFragment(...): Encountered an invalid child; DOM "+"elements are not valid children of React components.");var result=[];for(var key in object){if(false){}mapIntoWithKeyPrefixInternal(object[key],result,key,emptyFunction.thatReturnsArgument)}return result}module.exports=createReactFragment},9:function(module,exports,__webpack_require__){"use strict";var validateFormat=function validateFormat(format){};if(false){}function invariant(condition,format,a,b,c,d,e,f){validateFormat(format);if(!condition){var error;if(format===undefined){error=new Error("Minified exception occurred; use the non-minified dev environment "+"for the full error message and additional helpful warnings.")}else{var args=[a,b,c,d,e,f];var argIndex=0;error=new Error(format.replace(/%s/g,function(){return args[argIndex++]}));error.name="Invariant Violation"}error.framesToPop=1;throw error}}module.exports=invariant}});
```

Comment:
I don't believe you need to commit this file.

Reply:
Removed!